frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -51,7 +52,7 @@ if (Cypress.env('BRIDGE_AWS')) {
         `snapshot.storage.k8s.io~v1beta1~VolumeSnapshot/${PVC.metadata.name}-snapshot`,
       );
       detailsPage.titleShouldContain(PVC.metadata.name);
-      resourceStatusShouldContain('Ready');
+      waitForResourceStatusToContain('Ready');
```

Comment:
https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/cypress.json#L13
We already wait for 30 seconds which is enough.